kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/utils/KoPZKUtils.java

```diff
@@ -0,0 +1,47 @@
+package io.streamnative.pulsar.handlers.kop.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.zookeeper.CreateMode;
+import org.apache.zookeeper.ZooDefs;
+import org.apache.zookeeper.ZooKeeper;
+import org.apache.zookeeper.data.Stat;
+
+/**
+ * Utils for ZooKeeper.
+ */
+@Slf4j
+public class KoPZKUtils {
+    public static void createPath(ZooKeeper zooKeeper, String zkPath, String subPath, byte[] data) {
```

Comment:
Hi @dockerzhang what is the reason to add this utils?

Reply:
when brokers handle fetching requests,  there is no `groupId` in the request header or data, we could store the  `groupId` in `zookeeper` for the current connection client.